cluster-autoscaler/cloudprovider/azure/azure_manager.go

```diff
@@ -148,31 +148,72 @@ type Config struct {
 }
 
 // InitializeCloudProviderRateLimitConfig initializes rate limit configs.
-func InitializeCloudProviderRateLimitConfig(config *CloudProviderRateLimitConfig) {
+func InitializeCloudProviderRateLimitConfig(config *CloudProviderRateLimitConfig) error {
```

Comment:
Thanks for adding this. Would you like to add some unit tests for the changes?

Reply:
@feiskyer I've added some unit tests in https://github.com/kubernetes/autoscaler/pull/3114/commits/1d6f18ff3cb577b0c1f476c59359d75d20324e25

Reply:
thanks a lot